pom.xml

```diff
@@ -122,6 +126,16 @@
       </plugin>
     </plugins>
     </pluginManagement>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>1.8</source>
+          <target>1.8</target>
```

Comment:
Yes, use those.  I will come along later and move the project to a parent pom that is not redefining the standard properties.